templates/careers/careers_base.html

```diff
@@ -88,7 +88,7 @@
       </select>
     </div>
     <div class="col-9 col-start-large-5">
-      {% if (request.path != "/careers/lifestyle" and request.path != "/careers/ethics" and request.path != "/careers/travel" and request.path != "/careers/progression" and request.path != "/careers/diversity" and request.path != "/careers/job-detail") %}
+      {% if (request.path != "/careers/lifestyle" and request.path != "/careers/ethics" and request.path != "/careers/travel" and request.path != "/careers/progression" and request.path != "/careers/diversity" and request.path != "/careers/jobs/<path>") %}
```

Comment:
Didn't you change the `/careers/jobs` path?

Reply:
Anyway, this conditional be neater with something like:

``` python3
{% if request.path not in ["/careers/lifestyle", "/careers/ethics", ...] %}
```

Reply:
Yes, I did, but forgot to change here.